packages/swapper/src/swappers/cow/utils/cowService.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+const axiosConfig = {
+  baseURL: 'https://api.cow.fi/mainnet/api/',
```

Comment:
Does cowswap support any other networks other than mainnet? If so, it may be a good idea to pass the baseUrl as a dep so it can be set by web instead of hard coding it here. It may be out of the scope of this PR (given this is how zrxSwapper does it), but it might be worth looking at.

here is a reference of how it's being done in [thorchainSwapper](https://github.com/shapeshift/lib/blob/5cd7af8106c3d7e3bc619e993d2f1a85ba4f4a11/packages/swapper/src/swappers/thorchain/utils/getUsdRate/getUsdRate.ts#L25)

Reply:
Indeed, there are other networks supported. I did similar thing as in thorchainSwapper.